presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,6 +171,55 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules govern who is considered an owner of a schema. The user is granted
```

Comment:
Schema rules allow you to grant ownership of a schema.

Reply:
As a next sentence we should document what ownership of a schema means..

Reply:
Changed to: 

> These rules allow you to grant ownership of a schema. Having ownership of an
> schema allows users to execute `DROP SCHEMA`, `ALTER SCHEMA` (both renaming
> and setting authorization), `SHOW CREATE SCHEMA`.